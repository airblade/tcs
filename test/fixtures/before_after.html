<button class="text-white px-4 sm:px-8 py-2 sm:py-3 bg-sky-700 hover:bg-sky-800">...</button>
