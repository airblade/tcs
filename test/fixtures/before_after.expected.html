<button class="bg-sky-700 px-4 py-2 text-white hover:bg-sky-800 sm:px-8 sm:py-3">...</button>
